### PR TITLE
allow achievements to show additional info when unlocked

### PR DESCRIPTION
**Pull request by KPCCoiL**
リアクションが付いた時に該当メッセージへのリンクをつけるなど用です

### CODE REVIEW

**Comment by niconegoto** [comment]
# [Codecov](https://codecov.io/gh/tsg-ut/slackbot/pull/241?src=pr&el=h1) Report
> Merging [#241](https://codecov.io/gh/tsg-ut/slackbot/pull/241?src=pr&el=desc) into [master](https://codecov.io/gh/tsg-ut/slackbot/commit/6156d83eb50e2079b4cf39bf792ef39ccde6d492&el=desc) will **increase** coverage by `0.00%`.
> The diff coverage is `n/a`.

[Impacted file tree graph](https://codecov.io/gh/tsg-ut/slackbot/pull/241?src=pr&el=tree)

```diff
@@           Coverage Diff           @@
##           master     #241   +/-   ##
=======================================
  Coverage   27.61%   27.61%           
=======================================
  Files          84       84           
  Lines        6898     6901    +3     
  Branches     1327     1330    +3     
=======================================
+ Hits         1905     1906    +1     
- Misses       4355     4356    +1     
- Partials      638      639    +1     
```


| [Impacted Files](https://codecov.io/gh/tsg-ut/slackbot/pull/241?src=pr&el=tree) | Coverage Δ | |
|---|---|---|
| [prime/index.js](https://codecov.io/gh/tsg-ut/slackbot/pull/241/diff?src=pr&el=tree#diff-cHJpbWUvaW5kZXguanM=) | `17.79% <0.00%> (+0.28%)` | :arrow_up: |

------

[Continue to review full report at Codecov](https://codecov.io/gh/tsg-ut/slackbot/pull/241?src=pr&el=continue).
> **Legend** - [Click here to learn more](https://docs.codecov.io/docs/codecov-delta)
> `Δ = absolute <relative> (impact)`, `ø = not affected`, `? = missing data`
> Powered by [Codecov](https://codecov.io/gh/tsg-ut/slackbot/pull/241?src=pr&el=footer). Last update [6156d83...9fa846a](https://codecov.io/gh/tsg-ut/slackbot/pull/241?src=pr&el=lastupdated). Read the [comment docs](https://docs.codecov.io/docs/pull-request-comments).

File: achievements/index_production.ts [hunk 1]
```diff
@@ -309,7 +312,7 @@ export const unlock = async (user: string, name: string) => {
 			<@${user}>が実績【${achievement.title}】を解除しました:tada::tada::tada: <https://sig.tsg.ne.jp/achievement-viewer/users/${user}|[実績一覧]>
 			_${achievement.condition}_
 			難易度${difficultyToStars(achievement.difficulty)} (${achievement.difficulty}) ${isFirst ? '*初達成者!!:ojigineko-superfast:*' : ''}
-		`,
+			${additionalInfo == null ? '' : additionalInfo}`,
```
Comment (hakatashi) [inline achievements/index_production.ts]: === undefined でおなしゃす！

Comment (KPCCoiL) [reply achievements/index_production.ts]: 自分でも存在忘れてましたけど直しました

**Comment by KPCCoiL** [comment]
わーい

**Pull request merged**